examples/classification/plot_classification_probability.py

```diff
@@ -54,42 +79,66 @@
     "GPC": GaussianProcessClassifier(kernel),
 }
 
+
+# %%
+# Plot classification probabilities for each classifier
+# -----------------------------------------------------------
+# In this section, we train multiple classifiers and then display their decision boundaries
+# and classification probabilities to compare their performance on the Iris data.
+
 n_classifiers = len(classifiers)
 
+# Set the size and adjust the subplot positions of the figure
 plt.figure(figsize=(3 * 2, n_classifiers * 2))
 plt.subplots_adjust(bottom=0.2, top=0.95)
 
+# Create a meshgrid for the decision boundary visualization
```

Comment:
No need for this comment as well. I open a PR to improve the support for the `DecisionBoundaryDisplay`: https://github.com/scikit-learn/scikit-learn/pull/26995. We should be able to remove the code once this PR is merged.